test/unit/depricated/dse-auth.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+const assert = require("assert");
+const DsePlainTextAuthProvider = require("../../../lib/datastax/depricated-auth/dse-plain-text-auth-provider");
+const DseGssapiAuthProvider = require("../../../lib/datastax/depricated-auth/dse-gssapi-auth-provider");
+
+describe("DsePlainTextAuthProvider", function () {
+    it("tests not supported error", function () {
+        assert.throws(function () {
+            new DsePlainTextAuthProvider();
+        }, ReferenceError);
```

Comment:
Those tests work because a reference error is thrown (``ReferenceError: throwNotSupported is not defined``) but not the one that should be here. Check the error message in at least one place.